x-pack/libbeat/management/api/enroll_test.go

```diff
@@ -62,18 +62,49 @@ func TestEnrollValid(t *testing.T) {
 }
 
 func TestEnrollError(t *testing.T) {
-	beatUUID, err := uuid.NewV4()
-	if err != nil {
-		t.Fatal(err)
+	tests := []struct {
+		name         string
+		body         string
+		responseCode int
+	}{
+		{
+			name:         "invalid enrollment token",
+			body:         `{"message": "Invalid enrollment token"}`,
+			responseCode: 400,
+		},
+		{
+			name:         "invalid token response",
+			body:         `{"access_token": ""}`,
+			responseCode: 200,
+		},
 	}
```

Comment:
In general I'm in favor of `map[string]struct{ ... }`. This way tests order is randomized. Just so to be sure we have no implicit dependencies on test order.

Reply:
I will make the change, good tip.